dictionaries/CallMap.php

```diff
@@ -5345,6 +5345,7 @@
 'imagealphablending' => ['bool', 'image'=>'GdImage', 'enable'=>'bool'],
 'imageantialias' => ['bool', 'image'=>'GdImage', 'enable'=>'bool'],
 'imagearc' => ['bool', 'image'=>'GdImage', 'center_x'=>'int', 'center_y'=>'int', 'width'=>'int', 'height'=>'int', 'start_angle'=>'int', 'end_angle'=>'int', 'color'=>'int'],
+'imageavif' => ['bool', 'image'=>'GdImage', 'file'=>'resource|string|null', 'quality'=>'int', 'speed'=>'int'],
```

Comment:
`file`, `quality` and `speed` are optional
```suggestion
'imageavif' => ['bool', 'image'=>'GdImage', 'file='=>'resource|string|null', 'quality='=>'int', 'speed='=>'int'],
```